src/charm.py

```diff
@@ -1,33 +1,40 @@
 #!/usr/bin/env python3
-# Copyright 2020 Omnivector Solutions, LLC.
+# Copyright 2024 Omnivector, LLC.
```

Comment:
[nit]: Copyright year should be 2020-2024 since the file was first added in 2020 :smile: 

```suggestion
# Copyright 2020-2024 Omnivector, LLC.
```